presto-spark-base/src/main/java/com/facebook/presto/spark/PrestoSparkSessionProperties.java

```diff
@@ -60,9 +60,15 @@
     public static final String SPARK_RESOURCE_ALLOCATION_STRATEGY_ENABLED = "spark_resource_allocation_strategy_enabled";
     public static final String SPARK_RETRY_ON_OUT_OF_MEMORY_HIGHER_PARTITION_COUNT_ENABLED = "spark_retry_on_out_of_memory_higher_hash_partition_count_enabled";
     public static final String SPARK_HASH_PARTITION_COUNT_SCALING_FACTOR_ON_OUT_OF_MEMORY = "spark_hash_partition_count_scaling_factor_on_out_of_memory";
+    public static final String ADAPTIVE_PICK_JOIN_SIDES_ENABLED = "adaptive_pick_join_sides_enabled";
```

Comment:
adaptive_join_order_enabled/adaptive_join_side_enabled/adaptive_join_side_switching_enabled ?

Reply:
sure- i changed it to adaptive_join_side_switching_enabled.